client/src/assets/sidebar-icons.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react"
+
```

Comment:
How about make all icons to be in a single file, and this file should not under assets